components/annot-columns.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <b-card no-body>
+    <!--    TODO: make this also toggleable like the explanation tab-->
+    <b-card-header>Review the annotated columns</b-card-header>
+    <b-card-body style="height:30vh; overflow-y:scroll;">
```

Comment:
Move to style tag below for this class or create an id for this element to reference in CSS